config/environments/test.rb

```diff
@@ -62,4 +62,9 @@
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
+
+  # Chrome (including headless) blocks cookies with SameSite=None (which we set
```

Comment:
Can you add something about Chrome requiring `None ; Secure`, which we are unable to do in test.

Reply:
done